### ISSUE TITLE
Skipping heading on front page

### ISSUE THREAD

**Issue by jonathanbardo** [godaddy/wp-primer-theme#168]
In Primer we are currently skipping h3 headings which cause a problem of semantic and could confuse people with assistive technology devices.

A discussion should be started about what semantic to use for widget title.

<img width="1022" alt="screen shot 2017-02-24 at 8 22 17 am" src="https://cloud.githubusercontent.com/assets/1933681/23304942/66fad2f6-fa6a-11e6-8e4d-ced7495718a6.png">

https://wp-themes.com/primer

**Comment by aledesma-godaddy** [comment]
Closing issues and PRs in preparation for repository archival
